cypress/integration/app/settingsflow.spec.ts

```diff
@@ -137,4 +137,17 @@ describe("Settings flow", () => {
 
     cy.findByLabelText(/host expiry window/i).should("have.value", "5");
   });
+
+  it("if last email is Hello from Fleet", () => {
```

Comment:
I think this should remain in the same test as where we set up the SMTP configuration. https://docs.cypress.io/guides/references/best-practices#Having-tests-rely-on-the-state-of-previous-tests does a good job of explaining why we would like to do this.

Reply:
@zwass 

"remain in the same test as where we set up the SMTP configuration" -- what file is that?

I went ahead and made changes from your last comments, let me know if I've missed the mark on anything!